File: java/src/atcoder/ABC103/C.java
package atcoder.ABC103;

import java.util.Scanner;

public class C {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();

        // mについての制限はないので、
        // 各余りの最大値はそれぞれ(Ai-1)と考えて良い
        // そのようなmを選んだと仮定すればよい(例えば全ての積から-1したもの)

        long ans = 0;
        for (int i = 0; i < n; i++) {
            ans += sc.nextInt() - 1;
        }

        System.out.println(ans);
    }

}

File: kotlin/src/atcoder/ABC103/C.kt
package atcoder.ABC103

fun main(args: Array<String>) {
    val n = readLine()!!
    val xs = readLine()!!.split(" ").map(String::toInt)

    // mについての制限はないので、
    // 各項の余りが最大(Ai-1)になるようなmを選んだとすればよい
    // ちなみに、そのようなmのうちのひとつには、各項の総乗から-1したものがある
    // つまり、各項の余りの最大値はAi-1なのでそれらの総和が答えになる

    println(xs.map { it - 1 }.sum())
}